templates/project/src/main.rs

```diff
@@ -74,7 +74,7 @@ fn main() {
     }
 
     if !storage.has_robot_credentials() {
-        log::warn!("no machine configuration was found in storage");
```

Comment:
Both `esp32/main.rs` and `native/main.rs` have this as "machine credentials were". This is why I want to move this code out of the template.